tests/test_jointcal_decam.py

```diff
@@ -105,6 +105,53 @@ def test_jointcalTask_2_visits_constrainedAstrometry_no_photometry(self):
 
         self._testJointcalTask(2, relative_error, self.dist_rms_absolute, pa1, metrics=metrics)
 
+    def setup_jointcalTask_2_visits_constrainedPhotometry_no_astrometry(self):
+        """Help keep the constrainedPhotometry tests consistent and make
+        the differences between them more obvious.
+        """
+        self.config = lsst.jointcal.jointcal.JointcalConfig()
+        self.config.astrometryRefObjLoader.retarget(LoadAstrometryNetObjectsTask)
+        self.config.photometryRefObjLoader.retarget(LoadAstrometryNetObjectsTask)
+        self.config.photometryModel = "constrained"
+        self.config.sourceSelector['astrometry'].badFlags.append("base_PixelFlags_flag_interpolated")
+        self.config.doAstrometry = False
+        self.jointcalStatistics.do_astrometry = False
+
+        pa1 = 0.11
+        metrics = {'collected_photometry_refStars': 4865,
+                   'selected_photometry_refStars': 661,
+                   'associated_photometry_fittedStars': 6749,
+                   'selected_photometry_fittedStars': 2044,
+                   'selected_photometry_ccdImages': 14,
+                   'photometry_final_chi2': 3066.22,
+                   'photometry_final_ndof': 1998,
+                   }
+
+        return pa1, metrics
+
+    @unittest.skip("This test produces different chi2/ndof on Linux and macOS.")
```

Comment:
If there is a ticket to investigate and fix this discrepancy, could you make a note of it here? And if there isn't one yet, I suggest you create one.